website/docs/deployment.mdx

```diff
@@ -146,9 +146,9 @@ It is not the best option, compared to a static hosting provider / CDN.
 
 :::warning
 
-In the following sections, we will introduce a few common hosting providers and how they should be configured to deploy Docusaurus sites most efficiently. Some of the writeups are provided by external contributors. Docusaurus is not interest-related with any of the services. The documentation may not be up-to-date: recent changes in their API may not be reflected on our side. If you see outdated content, PRs are welcome.
+In the following sections, we will introduce a few common hosting providers and how they should be configured to deploy Docusaurus sites most efficiently. Docusaurus is not affiliated with any of these services, and this information is provided for convenience only. However, if you see outdated content, PRs are welcome.
```

Comment:
> Some of the writeups are provided by external contributors.
> The documentation may not be up-to-date: recent changes in their API may not be reflected on our side.

These are important caveats; why delete them?

Reply:
I was trying to be concise by covering them under the "this info is provided for convenience only/use at your own risk" umbrella, but if we want to be less vague, maybe this (changes in bold italics):

In the following sections, we will introduce a few common hosting providers and how they should be configured to deploy Docusaurus sites most efficiently. Docusaurus is not affiliated with any of these services, and this information is provided for convenience only. _**Some of the write-ups are provided by third-parties, and recent API changes may not be reflected on our side. If**_  you see outdated content, PRs are welcome.

Reply:
That is okay for me ^^

Reply:
Great, I'll make the change.